zebedee-cms/pom.xml

```diff
@@ -246,6 +246,19 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-web</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Has this dependency only been added to  be able to use `MockHttpServletResponse`?

I think we could avoid introducing this dependency if we manually mocked it with Mockito. See [TeamsReportTest](https://github.com/ONSdigital/zebedee/blob/61095cad46af4eb8d66bb0e6de470e3de982b7bb/zebedee-cms/src/test/java/com/github/onsdigital/zebedee/api/TeamsReportTest.java#L102-L104)

Reply:
Yes, but that would have been more and more complex code in the test, and more brittle test cases. As the Spring-test  framework was already being used and in context, this was the simplest and most reliable way to build a HttpReq object and use it. Following the pattern of not doing what somebody else has done before, better and more reliably...

Reply:
Yes, `spring-test` is used but not `spring-web`. I don't really see this as a valid reason to include this additional dependency since this same behaviour can be achieved easily with Mockito (and in fact it is already done in other tests)

Reply:
We've removed spring-web